packages/api/amplify_api/lib/src/util.dart

```diff
@@ -30,3 +30,21 @@ Map<String, String>? addContentTypeToHeaders(
   modifiedHeaders.putIfAbsent(AWSHeaders.contentType, () => contentType);
   return modifiedHeaders;
 }
+
+/// Grabs errors from GraphQL Response. Is used in method channel and Dart first code.
+/// TODO: Move to Dart first code when method channel GraphQL implementation is removed.
```

Comment:
TODOs shouldn't be doc comments generally, and lets add usernames for all TODOs (this is a lint in the new packages, but this one hasn't been migrated yet)

```suggestion
// TODO(Equartey): Move to Dart first code when method channel GraphQL implementation is removed.
```